sonar-ws/src/main/protobuf/ws-components.proto

```diff
@@ -87,6 +87,21 @@ message SearchProjectsWsResponse {
   optional sonarqube.ws.commons.Facets facets = 3;
 }
 
+// WS api/components/provisioned
+message ProvisionedWsResponse {
+  repeated Component projects = 1;
+  optional int64 total = 2;
```

Comment:
This is the old paging way, the new way is to use : 
`optional sonarqube.ws.commons.Paging paging = 1;`
(you need to add another entry in the changelog)